src/data-base/GameData.js

```diff
@@ -0,0 +1,24 @@
+const GameModel = require("../model/GameModel");
+const InitialConnect = require("./SequelizeConnection");
+const ConfigConnect = require("../config/ConfigConnect.js");
+const instanceConnect = InitialConnect(ConfigConnect);
```

Comment:
"instanceConnect" is unclear, it should be "databaseConnection", "connection", "connectionInstance" or something like that

Reply:
the change was made according to the suggestion for connectionInstance